src/app/pages/details/phone-view/instructor-resources-pane/instructor-resources-pane.scss

```diff
@@ -41,7 +41,7 @@
             @include set-font(h4);
         }
 
-        .resource-box {
+        .Login to unlockx {
             width: 20rem;
```

Comment:
that doesn't seem like a valid selector?  maybe the result of a  bad search-and-replace?

Reply:
I don't know what happened there. Thanks for catching it!
Also, thanks for your prompt review of this behemoth.